openwisp_users/admin.py

```diff
@@ -142,6 +144,42 @@ class UserAdmin(MultitenantAdminMixin, BaseUserAdmin, BaseAdmin):
                     'last_login']
     inlines = [EmailAddressInline, OrganizationUserInline]
     save_on_top = True
+    actions = ['make_inactive', 'make_active']
+
+    def require_confirmation(func):
```

Comment:
nice solution!
Can you add a docstring indicating that this is a decorator function and what it does?
This will help future contributors and even ourselves.

Reply:
Can you please check the docstring I have added, I am not good at documenting my changes but tried to make it as simple as possible.